draft-iyengar-quic-delayed-ack.md

```diff
@@ -349,6 +327,43 @@ flight at any given time, this extension does not prohibit having more than one
 in flight. Generally, when using `max_ack_delay` for PTO computations, endpoints
 MUST use the maximum of the current value and all those in flight.
 
+# Implementation Considerations {#implementation}
+
+There are tradeoffs inherent in a sender sending an ACK_FREQUENCY frame to the
+receiver. As such it is recommended that implementers experiment with different
+strategies and find those which best suit their applications and networks.
+There are, however, noteworthy considerations when devising strategies for
+sending ACK_FREQUENCY frames.
+
+## Loss Detection {#loss}
+A sender relies on receipt of acknowledgements to determine the amount of data
+in flight and to detect losses, specifically when packets experience reordering,
+see {{QUIC-RECOVERY}}.  Consequently, how often a receiver sends acknowledgments
+determines how long it takes for losses to be detected at the sender.
```

Comment:
@ianswett : I don't think that's accurate, since the ack may be received before the time threshold period passes. I  prefer Matt's wording, since it's not necessary to wade into details here.

Reply:
Fair point Jana, ignore my suggestion Matt.